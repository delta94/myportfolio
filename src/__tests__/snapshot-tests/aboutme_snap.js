import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import AboutMe from '../../components/aboutme';

it('aboutme renders correctly', () => {
  const wrapper = shallow(<AboutMe />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
