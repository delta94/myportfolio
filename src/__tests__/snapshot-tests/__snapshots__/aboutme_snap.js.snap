// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aboutme renders correctly 1`] = `
<div
  className="box"
  style={
    Object {
      "backgroundColor": "transparent",
      "boxShadow": "none",
      "position": "relative",
    }
  }
>
  <styled.div
    className="content"
  >
    <p>
      Hey I'm Prabhu Murthy
    </p>
    <p>
      I'm a
      <styled.span>
        fullstack
      </styled.span>
      javascript developer, designer and very passionate about developing apps that scale very well.
    </p>
    <p>
      My Core expertise is building frontend apps through 
      <styled.span>
        React/Vue
      </styled.span>
      and build app servers using Node
      <styled.span>
        (Express/Sails).
      </styled.span>
    </p>
    <p>
      I have extensive experience in designing and building web apps that are 
      <styled.span>
        highly secure & fast performing.
      </styled.span>
    </p>
  </styled.div>
</div>
`;
